draft-irtf-cfrg-hpke.md

```diff
@@ -830,6 +829,20 @@ security models and assumptions, and do not consider attackers capable of quantu
 computation. A full proof of post-quantum security would need to take this
 difference into account, in addition to simply using a post-quantum KEM.
 
+## Pre-Shared Key Recommendations
+
+In the PSK mode (see {{mode-psk-auth}}), PSKs SHOULD be at least Nh bytes long,
+and SHOULD be derived from at least Nh bytes of entropy.  While an HPKE
+ciphertext does not reveal anything about the PSK (as a consequence of the
+IND-CCA2 property), application behaviors may create an oracle that tells an
+attacker whether a given PSK is correct or not.  For example, a receiver
+whose behavior is observably different based on the success or failure of an
+HPKE decryption would provide such an oracle.  Given such an oracle, an
+adversary may be able to recover a low-entropy PSK by brute-force search.
+
+This risk does not arise for the AuthPSK mode, because the attacker cannot
+generate candidate ciphertexts without access to the sender's private key.
```

Comment:
I tried to find a more general formulation in #50 that considers all cases where the adversary has access to zz.

Reply:
👍  I left a comment over there.